webapp/api/launchpad.py

```diff
@@ -0,0 +1,29 @@
+from webapp import api
```

Comment:
as mentionned in chat. Can we use the launchpadlib to avoid depending of self written API files.